PR: Move some preferences entries to their own modules

ccordoba12 opened the pull request:
This PR makes some refactoring of our preferences to make things easier to understand for newcomers.

pep8speaks commented:
Hello @ccordoba12! Thanks for submitting the PR.



 - In the file [`spyder/preferences/appearance.py`](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/appearance.py), following are the PEP8 issues :

> [Line 398:44](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/appearance.py#L398): [E127](https://duckduckgo.com/?q=pep8%20E127) continuation line over-indented for visual indent
> [Line 400:44](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/appearance.py#L400): [E127](https://duckduckgo.com/?q=pep8%20E127) continuation line over-indented for visual indent
> [Line 407:1](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/appearance.py#L407): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> [Line 413:1](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/appearance.py#L413): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> [Line 418:80](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/appearance.py#L418): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (81 > 79 characters)
> [Line 419:1](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/appearance.py#L419): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace



 - In the file [`spyder/preferences/general.py`](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/general.py), following are the PEP8 issues :

> [Line 116:80](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/general.py#L116): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (80 > 79 characters)
> [Line 117:50](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/general.py#L117): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 117:52](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/general.py#L117): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 117:80](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/general.py#L117): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (83 > 79 characters)
> [Line 220:19](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/general.py#L220): [W605](https://duckduckgo.com/?q=pep8%20W605) invalid escape sequence '\.'
> [Line 220:39](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/general.py#L220): [W605](https://duckduckgo.com/?q=pep8%20W605) invalid escape sequence '\.'
> [Line 221:33](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/general.py#L221): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 222:33](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/general.py#L222): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 226:80](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/general.py#L226): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (81 > 79 characters)
> [Line 273:17](https://github.com/spyder-ide/spyder/blob/023b690127f597572e9732d1326b44b2b9af20bb/spyder/preferences/general.py#L273): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent

goanpeca commented:
@ccordoba12 could you post a couple of screenshots :-p ?

ccordoba12 commented:
Nothing changes here. I just moved `MainConfigPage` and `ColorSchemeConfigPage` to `preferences/general.py` and `preferences/appearance.py` respectively (and renamed the last one to `AppearanceConfigPage`).

goanpeca commented:
@ccordoba12 ah ok :-)
